app/src/pages/prompt/PromptVersionsList.tsx

```diff
@@ -35,10 +37,15 @@ const promptVersionItemCSS = css`
  * add /:versionId to the current path
  */
 export const PromptVersionItem = ({ version }: PromptVersionItemProps) => {
+  const { versionId } = useParams();
   const navigate = useNavigate();
+  const active = versionId === version.id;
   return (
     <div css={promptVersionItemCSS}>
-      <Button onClick={() => navigate(`${version.id}`)} variant="quiet">
+      <Button
+        onClick={() => navigate(`${version.id}`)}
+        variant={active ? "default" : "quiet"}
+      >
```

Comment:
I love that you made this accessible! however it probably shouldn't rely on style overloading. I would make it a raw button and just style it simply.

Reply:
Replaced with styled Link/Anchor